src/index.ts

```diff
@@ -34,6 +34,7 @@ const CompletionItemKindMap = Object.fromEntries(
 
 const useLast = (values: readonly any[]) => values.reduce((_, v) => v, '');
 
+const transport = Facet.define<Transport, Transport>({ combine: useLast });
```

Comment:
I think we can get rid of this. Transport lives as a property on the LanguageServerClient object. We don't quite need a CodeMirror facet for it.